### ISSUE TITLE
[Feature Request] Change log for modified value.

### ISSUE THREAD

**Issue by croshine** [snipe/snipe-it#2596]
I wish I could see the change log for modified value of each field.

For example, If someone changed the location of an asset,
another user can check the change log in history tab like below.

**`Location`** has been changed by **`jmbae`** at 2016/09/09 15:30:24 : **`1st Storage`** → **`Server Room`**.

This feature may allows to track someone's input mistake or malicious data modification.

**Comment by grandmamoses** [comment]
Hi

I vote for this too. Attribute changes, especially location and who made the change are very important to us. These being saved to the history would be great.

Thanks!

**Comment by ultimatee** [comment]
Voting for this too. With this feature it would be possible for example to track the turnover at the end of the month.

**Comment by matc** [comment]
Up voting this too. Having a record of who an asset was assigned to historically is important to us also so having the feature natively in SnipeIT would be great. 

Thank you!

**Comment by jasonturner42** [comment]
Throwing a vote in for this as well.  Ideally we'd be able to drill down on the changes as well, but a report that shows the changes would suffice as well for us.

**Comment by snipe** [comment]
@matc 

> Having a record of who an asset was assigned to historically is important to us also so having the feature natively in SnipeIT would be great.

That's literally what the history tab is for though.

**Comment by matc** [comment]
Hey @snipe thanks for responding so quickly. 

What led to my searching for this was we noticed when an asset was updated, the history tab would say the asset was updated by who and when, but nothing more. For example, if the location changed, this is not captured.  
After seeing your comment today, I noticed when an asset is checked-in/checked-out - the history of who it was previously assigned to along with any custom notes added do show up. This is great and means we should be able to move forward with our roll-out but is a change from what we're used to. 

Does that  sounds like expected behavior or have we maybe messed up something with our install?  If not, would it be a big level of effort to add more detail to those history entries when assets are simply updated?

Thanks very much for all you do!
